docs/nodes/maintain/avalanchego-config-flags.md

```diff
@@ -499,6 +499,7 @@ Specifies where to find AvalancheGo & plugin binaries. Defaults to the path of e
 ```text
 build-dir
 |_avalanchego
+  |_build
     |_plugins
       |_evm
```

Comment:
This isn't right, but the suggested change isn't right, either.

It should be:

```
build-dir
|_avalanchego
  |_plugins
    |_evm
```
That is: underneath the build directory, there is the `avalanchego` binary and the `plugins` directory. Under the `plugins` directory is the `evm` binary.

Reply:
I think this suggested change conflates the `avalanchego` binary (what is intended here) with a directory named `avalanchego` (which isn't relevant in this context.)